test/plugin/base_test.rb

```diff
@@ -605,6 +605,63 @@ def test_sanitize_tags_from_container_name_with_require_valid_tags_false
     end
   end
 
+  def test_configure_split_logs_by_tag
+    setup_gce_metadata_stubs
+    {
+      APPLICATION_DEFAULT_CONFIG => true,
+      DISABLE_SPLIT_LOGS_BY_TAG_CONFIG => false
+    }.each do |(config, split_logs_by_tag)|
+      d = create_driver(config)
+      assert_equal split_logs_by_tag,
+                   d.instance.instance_variable_get(:@split_logs_by_tag)
+    end
+  end
+
+  def test_split_logs_by_tag
+    setup_gce_metadata_stubs
+    log_entries_count = 5
+    dynamic_log_names = Array(0..log_entries_count - 1).map do |index|
```

Comment:
I was using `log_entries_count.times.map` but rubocop does not like it...

```
C: Use Array.new with a block instead of .times.map
```

`(0..log_entries_count - 1).map` seems to work though. Just changed to that.